src/components/Posts/Posts.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styles from './Posts.module.css';
+import {PostsData } from '../../Api/PostsApi.ts';
+import Post from '../Post/Post.tsx';
+import { PostType } from '../../DataTypes/PostType.ts';
```

Comment:
you can order the imports to be 
react libraries
other extrenal libraries
components
utils/ helpers/ constants files
styles